packages/radix-ui-themes/src/components/callout.tsx

```diff
@@ -55,16 +55,8 @@ type CalloutIconElement = React.ElementRef<'div'>;
 interface CalloutIconProps extends ComponentPropsWithout<'div', RemovedProps> {}
 const CalloutIcon = React.forwardRef<CalloutIconElement, CalloutIconProps>(
   ({ className, ...props }, forwardedRef) => {
-    const { color, size, highContrast } = React.useContext(CalloutContext);
     return (
-      <Text
-        asChild
-        color={color}
-        size={mapResponsiveProp(size, mapCalloutSizeToTextSize)}
-        highContrast={highContrast}
-      >
-        <div {...props} className={classNames('rt-CalloutIcon', className)} ref={forwardedRef} />
-      </Text>
```

Comment:
Text seemed unnecessary here; it was really needed just for the line height values which make more sense in the CSS

Reply:
How is the size handled now?

Reply:
See the CSS changes in callout.css, it's just one variable